lasy/utils/mode_decomposition.py

```diff
@@ -9,7 +9,15 @@
 from lasy.utils.exp_data_utils import find_d4sigma
 
 
-def hermite_gauss_decomposition(laserProfile, wavelength, m_max=12, n_max=12, res=1e-6):
+def hermite_gauss_decomposition(
+    laserProfile,
+    wavelength,
+    m_max=12,
+    n_max=12,
+    res=1e-6,
+    lo=[-2e-4, -2e-4],
+    hi=[2e-4, 2e-4],
```

Comment:
Could you make these 3 mandatory parameters?

Reply:
Thanks you for the feedback. The changes suggested have been made.

Reply:
Thank you for the feedback:)
The requested changes have been made. Additionally n_max and m_max also needed to be made mandatory in order to keep the code as consistent as possible.